README.md

```diff
@@ -10,33 +11,57 @@ This project is a living example of a client app you could start building with T
 To get started, clone the repository, change directory and install the dependencies:
 
 ```shell
-git clone git@github.com:kiwicom/margarita.git
+https://github.com/kiwicom/margarita
```

Comment:
Why not `git clone git@github.com:kiwicom/margarita.git` ?

Reply:
`https` doesn't need any authentication which seems to me better in this case. But it's not a big deal, we can keep the old one.

Reply:
Ok, thanks, that makes sense. But then it should be at least `git clone https://github.com/kiwicom/margarita`

Reply:
I see, you are right! Thank you. It's copy-paste issue

Reply:
Now it should be fixed